.vscode/settings.json

```diff
@@ -45,6 +45,7 @@
   },
   "editor.rulers": [120],
   "eslint.workingDirectories": [{ "mode": "auto" }], // infer working directory based on .eslintrc/package.json location
+  "eslint.validate": ["json"],
```

Comment:
added by nx migration - enables to write lint rules for json as well in the future